README.md

```diff
@@ -22,16 +22,7 @@ Official Starting Page: https://dotnet.microsoft.com/
 
 We welcome contributions! Many people all over the world have helped make this project better.
 
-* If you have a question or idea, [file a new issue](https://github.com/dotnet/runtime/issues/new).
-
-	* Is your feedback for .NET Framework? This repo is for .NET Core. If you are having issues with the "full" .NET Framework (also called "Desktop"), the best way to file a bug is the [Report a Problem](https://aka.ms/vs-rap) tool, which is integrated with the [VS Developer Community Portal](https://developercommunity.visualstudio.com/); or through [Product Support](https://support.microsoft.com/en-us/contactus?ws=support) if you have a contract.
-
-* If you want to contribute a fix or improvement, you'll want to check out these links
-
-    * Requirements to build this repo ([Linux](docs/building/linux-requirements.md), [macOS](docs/building/macos-requirements.md), [Windows](docs/building/windows-requirements.md))
```

Comment:
Are we sure we want to delete this entire section? The `Contributing.md` doc doesn't have instructions to build the repo. Maybe delete the `If you have a question or idea...` section but keep the `If you want to contribute a fix...` section?

Reply:
yes we should delete this and in CONTRIBUTING.md point to the build instructions: https://github.com/dotnet/runtime/blob/master/docs/workflow/README.md

Reply:
@gfoidl : Do you mind if I add a commit to this patch updating some more outdated links?